ui/raidboss/data/07-dt/eureka/occult_crescent_south_horn.txt

```diff
@@ -144,9 +116,6 @@ hideall "--sync--"
 # IGNORED ABILITIES
 # A160 Probably an auto attack
 #
-# TODO
```

Comment:
You probably wanted to keep this `TODO` comment line, since the `Figure out which timeline entries` item is still listed.

Reply:
Right. I only meant to get rid of the question not the TODO. Fixed.